### PR TITLE
sdl2: new package at 2.0.10

### PR DESCRIPTION
**Pull request by alaughlin**
## Description of package
https://www.libsdl.org/index.php

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/sdl2/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+patch -p1 < SDL2-2.0.10-opengl_include_fix-1.patch
```
Comment (alaughlin) [inline community/sdl2/build]: note: LFS seems to think this is a required patch for i686 systems: http://www.linuxfromscratch.org/blfs/view/svn/multimedia/sdl2.html

if you don't think we need it, I'll remove it

Comment (dylanaraps) [reply community/sdl2/build]: We don't support i686 systems, remove it. :+1:

Comment (alaughlin) [reply community/sdl2/build]: removed!

**Comment by dylanaraps** [comment]
Thanks!

**Pull request merged**